src/components/scene/postprocessing.js

```diff
@@ -0,0 +1,5 @@
+var register = require('../../core/component').registerComponent;
```

Comment:
We can just take it out, the post processing logic is triggered with .getAttribute("postprocessing") so the component is in fact not needed